api/src/lib/sqs.ts

```diff
@@ -1,24 +1,21 @@
 import AWS_SQS, { SQS } from '@aws-sdk/client-sqs';
-import * as Helpers from './helpers';
 
-const queueUrl = Helpers.checkEnvVariable('QUEUE_URL');
-const endpoint = Helpers.checkEnvVariable('SQS_ENDPOINT');
+const queueUrl = process.env.QUEUE_URL;
```

Comment:
Do you still need to check the environment variable exists? What will happen now if the code can't find the variable?

Reply:
Instead of throwing an error when it tries to access the variables on lines 4+5, it would go further and throw an error when it can't connect to the queue properly when it creates the client I think. I feel like it is better with the helper now... I'll put it back

Reply:
Probably is better, it will give you a clearer error to diagnose.